up-l1/mqtt_5.adoc

```diff
@@ -235,5 +235,10 @@ When MQTT5 (broker) is used for D2D communication, the topic shall consist of on
 ===== Sending a Message
 `UTransport::send(UMessage)` translates to the MQTT publish to topic `{UMessage.source.authority_name}/{UMessage.sink.authority_name}`
 
+== MQTT5 Connection handling
 
+=== Establishing a connection
+Establishing a connection to an MQTT broker *SHOULD* be done with `cleanSession = true` and `sessionExpiry = 0`. With those settings the broker will not buffer any messages while a the client is offline and will loose any subscriptions on a client disconnect. 
```

Comment:
We actually might want to buffer messages in the broker for the scenario so vehicles can receive messages when they wakeup so I'm not sure if we want to make this a SHOULD. I think the transport needs to have a configuration option for this setting.

Reply:
Ok, I will rewrite it in a form that we want the transport to have the options. IMHO going with "stateless" ie. cleanSession = true and sessionExpiry=0 is the best way for scaling because it opens up some possible deployment topologies but if you have state ie. sessions you are basically tied to a clustered MQTT deployment.

Reply:
Addressed here: https://github.com/jjj-vtm/up-spec/blob/d984eb005d26b15437f95337ada3c953c7687c09/up-l1/mqtt_5.adoc?plain=1#L245C1-L258C1